data/mods/gen9ssb/moves.ts

```diff
@@ -3920,6 +3920,33 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		type: "Water",
 	},
 
+	// za
+	shitpost: {
+		accuracy: 25500 / 256,
```

Comment:
use an onModifyAccuracy here

Reply:
did you mean onModifyMove? not sure i follow though, since the move's accuracy doesn't change from the base value.

Reply:
yes.